docs/pages/update-spec/introduction.md

```diff
@@ -0,0 +1,184 @@
+* `expo-sfv-version`  EAS Update version 0 MUST set this to `0`.
+* `expo-sfv`   [expo SFV 0](expo-sfv-0.md) version `0` is a partial implementation of Structured Field Values outlined in [IETF RFC 8941](https://tools.ietf.org/html/rfc8941)
+* `expo-manifest-filters` is an [expo SFV 0](expo-sfv-0.md) dictionary and is used to filter the updates stored by the client by `updateMetadata` attributes found in the [manifest](#manifest).
+  * For example: `expo-manifest-filters: branchname="main"` instructs the client to load the most recent update it has stored whose `updateMetadata` contains:
+
+  ```
+  updateMetadata: {
+    branchname: 'main',
+    ...
+  }
+  ```
+  * If the `branchname` manifest filter is included, it MUST equal the `branchName` in the `manifest.updateMetadata`.
+* `expo-server-defined-headers`  is an [expo-sfv](expo-sfv.md) dictionary. It is defines headers that a client MUST store and include in every subsequent [request](#request).
+
+  * For example, during a rollout we require a client to send back a stable token: `expo-server-defined-headers: expo-rollout-token="token". 
+* `cache-control` We recommend `cache-control: private, max-age=0`, but the only requirement is that it is a reasonable time frame. Please keep in mind that `updates` should be expected to be regularly created and the cache policy should not block this.
+
+### Manifest
+
+The body of the response MUST be a `manifest`. A `manifest` which is defined as a JSON conforming to the following description:
+
+Let an `Asset` be the JSON:
+```
+{
+  hash: string
+  key: string
+  contentType: string
+  url: string
+}
+```
+then the response body is referred to as the `manifest` and MUST be a json with format:
+```
+{
+  id: string
+  createdAt: date
+  runtimeVersion: string
+  launchAsset: Asset,
+  assets: Asset[],
+  updateMetadata: {
+    updateGroup: string
+    branchName: string
+  }
+}
+```
+  * `id` The ID MUST uniquely specify the manifest, however the different headers may accompany identical IDs in a response.
+  * `createdAt` Time created is essential as the client selects the most recent update (subject to any contraints supplied in the `expo-manifest-filters` header).
+  * `runtimeVersion` Can be any string defined by the developer. It stipulates what native code setup is required to run the associated javascript bundle.
+  * `Asset` Provides information about the asset and where to obtain it.
+    * `hash` SHA256 hash of the file to guarantee integrity.
+    * `key` Key used by the bundler.
+    * `contentType` e.g. `application/javascript`, `image/jpeg`.
+    * `url` Location where the asset is hosted.
+  * `launchAsset` The asset that is used to launch the app.
+  * `assets` assets used by the app, such as photos or fonts.
+  * `updateMetadata` The metadata associated with an update can be used for filtering the update (branchName) and also for the creation of more helpful errors.
```

Comment:
```suggestion
  * `updateMetadata` The metadata associated with an update. This can be used for filtering the update (branchName) and also for the creation of more helpful errors.
```

We should also explain the `updateGroup` and `branchName` sub keys here. 

> also for the creation of more helpful errors.

I think this should be removed or we should add a lot more explanation.

Reply:
Removed them.